extensions/chinese-lottery/README.md

```diff
@@ -0,0 +1,33 @@
+# Chinese Lottery ｜ 中国彩票
```

Comment:
Can we keep this README in english only?

Reply:
ok, It's only English now.